test/e2e/search_test.go

```diff
@@ -477,6 +477,6 @@ registries = ['{{.Host}}:{{.Port}}']`
 		search := podmanTest.Podman([]string{"search", "--limit", "130", "registry.redhat.io/rhel"})
 		search.WaitWithDefaultTimeout()
 		Expect(search).Should(Exit(0))
-		Expect(len(search.OutputToStringArray())).To(Equal(131))
+		Expect(len(search.OutputToStringArray())).To(BeNumerically(">", 100))
```

Comment:
This should check that this is not over 131 since the limit is set to 130.

Reply:
Much better :